### PR TITLE
Update RP-SAML.md

### PR DESCRIPTION
**Pull request by marcelodiiorio**
Removed various invalid "\\" characters in URLs.
Removed references to B2C Recorder, replacing it with Application Insights (see  "(Optional) Enable Debugging in your User Journey(s)")

### CODE REVIEW

File: Walkthroughs/RP-SAML.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ add the capability for your tenant to issue SAML tokens.
 > contains more details around each of the XML elements referenced in
 > this section.*
 
-1.  Open the B2C\_1A\_base.xml policy from your working directory.
+1.  Open the B2C_1A_base.xml policy from your working directory.
```
Comment (gsacavdm) [inline Walkthroughs/RP-SAML.md]: Please revert this change, this is needed to escape the underscore.

File: Walkthroughs/RP-SAML.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Add the SAML Relaying Party to User Journey(s)
 Now that your tenant can issue SAML tokens, we need to create a user
 journey that will be the one issuing this SAML tokens.
 
-1.  Open the B2C\_1A\_base.xml policy from your working directory.
+1.  Open the B2C_1A_base.xml policy from your working directory.
```
Comment (gsacavdm) [inline Walkthroughs/RP-SAML.md]: Please revert this change, this is needed to escape the underscore.

**Comment by marcelodiiorio** [comment]
Thanks for the feedback and clarifications, Gerardo. I will create a new pull request following your suggestions.
